crates/api_crud/src/post/create.rs

```diff
@@ -120,9 +121,12 @@ pub async fn create_post(
     )
   };
 
+  plugin_hook_mut("create_local_post", &mut post_form)?;
+
   let inserted_post = Post::create(&mut context.pool(), &post_form)
     .await
     .with_lemmy_type(LemmyErrorType::CouldntCreatePost)?;
+  plugin_hook("new_post", &inserted_post)?;
```

Comment:
Unlike comment, post only has a post-create hook?

Reply:
There is a pre hook right above.